package.json

```diff
@@ -10,7 +10,8 @@
     "test": "rm -rf ./fixtures && NODE_ENV=test node test",
     "test:watch": "NODE_ENV=test nodemon --ignore fixtures/ test",
     "format": "prettier-standard --format",
-    "cloc": "npx cloc lib/*.js cli.js html.js load.js"
+    "cloc": "npx cloc lib/*.js cli.js html.js load.js",
+    "build": "esbuild cli.ts --bundle --platform=node --outfile=cli.js"
```

Comment:
Perfect!

Reply:
We still need something like `build:watch` but i couldn't get that to work. For some reason when using `nodemon` it complains that it can't find ethe command `esBuild`

Reply:
Looks like we can use `--watch` from esbuild: https://esbuild.github.io/api/#watch

Reply:
Duh, I should read the docs better next time. Thanks!